### PR TITLE
Release 0.36.0

### PR DESCRIPTION
**Pull request by AnthonyOliveri**
Regenerated the SDK using:
- API Definitions [5d64e0df520d5620ab7bfd6cae9da917734918d7](https://github.ibm.com/Watson/developer-cloud--api-definitions/tree/5d64e0df520d5620ab7bfd6cae9da917734918d7)
- Watson Swagger Codegen [8dc2602de0f9555623a4721e51c48622a14cfdf5](https://github.ibm.com/MIL/watson-swagger-codegen/tree/8dc2602de0f9555623a4721e51c48622a14cfdf5)

This will be the first release using semantic-release!

Rather than lumping all generated changes into a single generation commit, plus a single "hand-edit" commit, I added and committed code strategically so that I could have a separate commitizen message for each update. This will be necessary moving forward so that the CHANGELOG and release notes are generated properly.

### CODE REVIEW

File: Source/DiscoveryV1/Models/QueryLarge.swift [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+
+/**
+ Object that describes a long query.
+ */
+public struct QueryLarge: Encodable {
```
Comment (mkistler) [inline Source/DiscoveryV1/Models/QueryLarge.swift]: This should have been generated as 'internal'.  Need to figure out why that didn't happen.

**Comment by AnthonyOliveri** [comment]
I changed `QueryLarge` to be `internal`, and I updated its `collectionIDs` property to be `String` instead of `[String]`.

**Comment by AnthonyOliveri** [comment]
Forgot to do this earlier: labeled as Do Not Merge until the Discover service is fixed in production, to allow `collectionIDs` to be passed as a comma-separated `String` instead of an `array`.

**Comment by AnthonyOliveri** [comment]
Just heard that the Discovery service fix is in production, so this can be merged as soon as it's approved.

**Pull request merged**